src/Ladybug_SunPath.py

```diff
@@ -509,8 +509,9 @@ def main(latitude, longitude, timeZone, elevation, north, hour, day, month, time
                     listInfo[i][5] = (1,1,1)
                     listInfo[i][6] = (12,31,24)
                     #find the numbers
-                    for h, hr in enumerate(sunUpHours):
-                        value = selList[int(math.floor(hr))] + (selList[int(math.ceil(hr))] - selList[int(math.floor(hr))])* (hr - math.floor(hr))
+                    for h, hr in enumerate(sunUpHours):                     
+#                        value = selList[int(math.floor(hr))] + (selList[int(math.ceil(hr))] - selList[int(math.floor(hr))])* (hr - math.floor(hr))
+                        value = selList[int(hr-1)]

```

Comment:
@devngc I assume this will only work for time-step of 1. The original code was written to support smaller time-steps such as every half an hour.